services/QuillLMS/app/assets/stylesheets/pages/science_of_reading_and_writing.scss

```diff
@@ -0,0 +1,338 @@
+.science-of-reading-and-writing-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  .horizontal-divider {
+    width: 200px;
+    height: 1px;
+    background-color: $quill-grey-25;
+  }
+  header .container {
+    img {
+      margin-bottom: 32px;
+    }
+    .text-section {
+      margin: 32px 0px;
+      text-align: center;
+      .first-line {
+        color: $quill-green;
+        text-align: center;
+        font-family: "Adelle Sans";
+        font-size: 32px;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 130%;
+        /* 41.6px */
+      }
+      h1 {
+        color: $quill-green;
+        font-family: "Adelle Sans";
+        font-size: 56px;
+        font-style: normal;
+        font-weight: 700;
+        line-height: 130%;
+      }
+    }
+    .description {
+      color: $quill-grey-50;
+      text-align: center;
+      font-family: "Adelle Sans";
+      font-size: 20px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 150%;
+      /* 30px */
+    }
+  }
+  .container {
+    padding: 96px 0px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    &.overview {
+      h2 {
+        text-align: center;
+        font-family: "Adelle Sans";
```

Comment:
I'm not sure about this, and totally defer to folks with more front-end expertise, but could some of these values be consolidated further up the logic tree?  For instance, could `font-family` be consolidated since I think it's the same value in every one of these declarations?  Or are these localized-ish overrides of our default?